receivers/line/line.go

```diff
@@ -42,7 +42,7 @@ func New(cfg Config, meta receivers.Metadata, template *templates.Template, send
 func (ln *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 	ln.log.Debug("executing line notification", "notification", ln.Name)
 
-	body := ln.buildMessage(ctx, as...)
+	body := ln.buildLineMessage(ctx, as...)
```

Comment:
can you add something like this below? 
https://github.com/grafana/alerting/blob/ec2c02267fa591e7252ddb44143555bc941ce6fa/receivers/telegram/telegram.go#L119-L127

Reply:
ahh the comment was about the limitation of message length! I applied this and rebased last commit 😃

Reply:
sorry for not being crystal clear!